applications/machine_learning/configuration/thingy53_nrf5340_cpuapp/prj.conf

```diff
@@ -179,6 +179,7 @@ CONFIG_LOG_BACKEND_UART=y
 # Enable USB CDC ACM
 
 CONFIG_USB_DEVICE_STACK=y
+CONFIG_USB_NRFX_WORK_QUEUE_STACK_SIZE=1200
```

Comment:
Doesn't it need to be fixed in Zephyr so all apps/samples benefit? If the stack size is too small?

Reply:
note that this should only affect nrf53 if there is such a need unless we know it affects other platforms too.

Reply:
@jfischer-no , what do you think?

Reply:
That surprises me very much. Let's not change it in upstream.